vector-stores/spring-ai-pgvector-store/src/main/java/org/springframework/ai/pg/vectorstore/PgVectorStore.java

```diff
@@ -628,11 +727,19 @@ public Builder withMaxDocumentBatchSize(int maxDocumentBatchSize) {
 		}
 
 		public PgVectorStore build() {
-			return new PgVectorStore(this.schemaName, this.vectorTableName, this.vectorTableValidationsEnabled,
-					this.jdbcTemplate, this.embeddingModel, this.dimensions, this.distanceType,
-					this.removeExistingVectorStoreTable, this.indexType, this.initializeSchema,
-					this.observationRegistry, this.searchObservationConvention, this.batchingStrategy,
-					this.maxDocumentBatchSize);
+			return PgVectorStore.builder(this.jdbcTemplate)
```

Comment:
Both `observationRegistry` and `searchObservationConvention` are missing in this builder setup?